### PR TITLE
Change color scheme of hero slide from white-on-green to dark-green-on-white

### PR DESCRIPTION
**Pull request by hoangviet1993**
Some static photos:
screencapture-localhost-8080-2018-09-09-08_55_41 1
screencapture-localhost-8080-2018-09-09-08_55_54
screencapture-localhost-8080-2018-09-09-08_56_06
screencapture-localhost-8080-2018-09-09-08_47_45

### CODE REVIEW

**Comment by hoangviet1993** [comment]
Changes are temporarily live at https://mail-app-214415.appspot.com/#!/.

File: static/pages/volunteer/volunteer.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h6 class="md-headline volunteer-page-title-box" show-xs hide-gt-xs>
       <div class="oppia-foundation-mobile-and-small-tablet-vertical-padding"
            layout="row"
            layout-align="center center">
-        <img src="/assets/images/volunteer/volunteer_page_global_graphics.png"
+        <img src="/assets/images/volunteer/Map.png"
```
Comment (seanlip) [inline static/pages/volunteer/volunteer.html]: Minor, but can we have our image filenames be all lowercase (for consistency)?

**Comment by hoangviet1993** [comment]
About page
screencapture-localhost-8080-2018-09-09-13_34_49

**Comment by hoangviet1993** [comment]
Waiting on designer feedbacks.

File: static/pages/volunteer/volunteer.html (hunk 1 above)
Comment (hoangviet1993) [reply static/pages/volunteer/volunteer.html]: Done!

**Pull request merged**